cmd/fyne_demo/screens/widget.go

```diff
@@ -201,11 +201,25 @@ func makeInputTab() fyne.Widget {
 
 func makeProgressTab() fyne.Widget {
 	progress = widget.NewProgressBar()
+
+	format := widget.NewProgressBar()
+	format.TextFormatter = func() string {
+		return fmt.Sprintf("%.2f out of %.2f", format.Value, format.Max)
+	}
+
+	go func() {
```

Comment:
Why not use the same loop as the other two progress bars?

Reply:
That's a good question. I honestly hadn't seen it. Thanks for pointing it out  :sweat_smile: